docs/sources/administration/grafana-server-cli/cli.md

```diff
@@ -1,6 +1,7 @@
 ---
 aliases:
   - administration/cli/
+  - cli/
```

Comment:
It's definitely worth adding a comment for the full URL to help in the future.

Also, note that relative aliases often need to be changed when the page moves since the relative destination changes too.

```suggestion
  - ../cli/ # /docs/grafana/latest/administration/cli/
  - ../../cli/ # /docs/grafana/latest/cli/
```